input/resources/au-patient.xml

```diff
@@ -103,21 +110,24 @@
     <element id="Patient.extension:genderIdentity">
       <path value="Patient.extension" />
       <sliceName value="genderIdentity" />
-      <short value="The patient's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
-      <comment value="Gender identity [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). See [Australian Bureau of Statics (ABS) Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) for further materials."/>
+      <short value="Gender identity shall be a member of the Gender Identity Response value set if any codes within that value set can apply"/>
+      <comment value="Individual Gender identity [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). See [Australian Bureau of Statics (ABS) Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) for further materials."/>
       <type>
         <code value="Extension" />
-        <profile value="http://hl7.org/fhir/StructureDefinition/patient-genderIdentity" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
       </type>
       <condition value="inv-pat-1"/>
     </element>
     <element id="Patient.extension:individualPronouns">
       <path value="Patient.extension"/>
       <sliceName value="individualPronouns"/>
+      <short value="Pronouns shall be a member of the Australian Pronouns value set if any codes within that value set can apply"/>
+      <comment value="Individual Pronouns [Australian Pronouns](http://terminology.hl7.org.au/ValueSet/au-pronouns) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))."/>
```

Comment:
wrong url for the value set - see qa.html for all the instances where this issue is present.

Reply:
This has been fixed, will push shortly.